Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -161,6 +161,33 @@ class CurrentUserUpdater: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: @escaping (Result<[BlockedUser], Error>) -> Void) {
```

Comment:
I don't see this being user anywhere? 🤔 It is not used in the controller nor the new state layer. 

Also, it would be nice to add to the Demo App, a way to show the blocked users, like we have with "Show Members"

Reply:
Updated 👍